### ISSUE TITLE
Harbor shows "-" for Version in about dialog

### ISSUE THREAD

**Issue by holzman** [bitnami/charts#7636]
**Which chart**:
bitnami/harbor

**Describe the bug**
Same as #5192: the "about" config displays "Version: -"

**Additional context**
This is probably just a peculiarity of how stacksmith is doing the build; the Makefile for harbor cats VERSION here:
https://github.com/goharbor/harbor/blob/master/Makefile#L165

and adds it to the binaries via a flag here:
https://github.com/goharbor/harbor/blob/master/Makefile#L169

**Comment by miguelaeh** [comment]
Thank you for reporting, I updated the internal task for the previous existing issue

**Comment by FraPazGal** [comment]
Hello @holzman,

This issue is now fixed in the latest chart's major, `12.0.0`. You can check it so by doing:

```
$ helm repo update
$ helm search repo bitnami/harbor
NAME          	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/harbor	12.0.0       	2.4.1      	Harbor is an open source trusted cloud-native r...
$ helm install harbor bitnami/harbor
```

<img width="246" alt="Screenshot 2022-03-09 at 13 43 32" src="https://user-images.githubusercontent.com/26460210/157444085-5bc7ca64-f912-4ff2-9e40-449ed6ee4add.png">

Thanks for reporting this bug and please do not doubt to reach out to us if there is any further issue.
